meson.build

```diff
@@ -447,6 +447,10 @@ endif
 
 nnstreamer_install_conf.set('ELEMENT_RESTRICTION_CONFIG', restriction_config)
 
+if get_option('install-test')
```

Comment:
if 'install-test' is false, TEST_TEMPLATE_DIR in ini file will not be updated.
plz remove this condition.

Reply:
Updated :)